src/common/components/ReviewModal/ReviewModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { Modal } from "@/common/components/Modal";
+import { type Review } from "@/common/components/ReviewItem";
+import { ProfileSchool } from "@/common/components/ReviewItem/ProfileSchool";
+import { ReviewBody } from "@/common/components/ReviewItem/ReviewBody";
+import ReviewHeader from "@/common/components/ReviewItem/ReviewHeader";
```

Comment:
question: would it make sense for these to be exported in `@/common/components/ReviewItem/index.ts`?

Reply:
good catch! fixed https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/79fb3d79570fd8a8455ef4e70dcc687af5bbf8b4